source/precalculus/source/08-TE/05.ptx

```diff
@@ -12,6 +12,464 @@
+        <answer>
+          <p>
+            The "angle of fall" is congruent to the same angle that is created with the base of the triangle (not the right angle). 
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Now that we know which angle is congruent to the "angle of fall" the pilot needs, which of the <m>6</m> trig functions could we use to find the angle at which she should descend?
+          </p>
+        </statement>
+        <answer>
+          <p>
+            If students label the triangle correctly, they should see that they can now use <m>\tan^{-1}</m> to find the measure of the angle because they have <m>40,000</m> as the opposite side and <m>100,000</m> as the adjacent side.
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Find the measure of the angle the pilot needs to make her initial descent.
+          </p>
+        </statement>
+        <answer>
+          <p>
+            Approximately <m>21.8</m><degree/>.
+          </p>
+        </answer>
+      </task>
+    </activity>
+
+    <remark>
+      <p>
+      Notice that <xref ref="right-triangles-1"/>, the angle that we needed to find was an angle that was not inside the right triangle. In these cases, it would be helpful to use prior knowledge of parallel lines and angle relationships to determine which other angle is congruent to that given angle.
+      </p>
+    </remark>
+    
+    <definition xml:id="def-angle-of-depression-and-elevation">
+      <statement>
+        <p>
+          Many applications of trigonometry will include the angle of elevation and the angle of depression which are formed by two parallel lines cut by a transversal. 
+          <ul>
+            <li>
+              <p>
+                An <term>angle of elevation</term> is the angle formed by a horizontal line and a line of sight to a point above the line.
+              </p>
+            </li>
+            <li>
+              <p>
+                An <term>angle of depression</term> is the angle formed by a horizontal line and a line of sight to a point below the line.
+              </p>
+            </li>
+          </ul>
+        </p>
+        <p> 
+          <figure xml:id="angle-of-elevation-and-depression">
+            <image source="Images/depression-elevation-angle.jpg"/>
+            <caption> The angle of elevation and the angle of depression are congruent.</caption>
+          </figure>
+        </p>
+        <p>
+          Notice that because both the angle of elevation and the angle of depression are formed by horizontal lines that are parallel, the angle of elevation is congruent to the angle of depression (by the alternate interior angles theorem).
+        </p>
+      </statement>
+    </definition>
+
+    <activity xml:id="right-triangles-2">
+      <introduction>
+        <p>
+          The Seattle Space Needle casts a <m>67</m>-meter shadow.
```

Comment:
This sentence probably needs some kind of acknowledgement that this number is not constant, and happened at a fixed point in time!  In fact, it may be better to not use shadow at all here, and just have an observer standing 67m away and noting a 70 degree angle of eleveation..